src/account.ts

```diff
@@ -341,8 +415,21 @@ export abstract class Account {
         contractFactory: StarknetContractFactory,
         options: DeclareOptions = {}
     ): Promise<string> {
+        let maxFee = options?.maxFee;
+        if (maxFee && options?.overhead) {
+            const msg = "Both maxFee and overhead cannot be specified";
```

Comment:
This is a misleading message, sounds like neither maxFee nor overhead can be specified. Perhaps better wording would be `maxFee and overhead cannot be specified together` or `maxFee and overhead cannot both be specified`

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/293/commits/672522ac4a38bc9e70bdfd80a54a6fd9852da162